changelog/fix-8040-new-account-waiting-period-notice

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+On Payments Overview, show the first deposit waiting period notice for new accounts that have transacted
```

Comment:
Could use `reinstate` or similar wording to clarify that this is putting something back. 

e.g. `Reinstate first deposit waiting period notice in payments overview (fix bug)`

Reply:
😃 resolved in 721c3c6faeb27e19bdfbcedc0f9f75dad664e019